content/src/service/synchronization/EventDeployer.ts

```diff
@@ -186,9 +186,9 @@ export class EventDeployer {
     }
 
     /** Execute an operation on the cluster, but return 'undefined' if it fails */
-    private async tryOnClusterOrUndefined<T>(execution: (server: ContentServerClient) => Promise<T>, cluster: ContentCluster, options?: { retries?: number, preferred?: ContentServerClient}): Promise<T | undefined> {
+    private async tryOnClusterOrUndefined<T>(execution: (server: ContentServerClient) => Promise<T>, cluster: ContentCluster, description: string, options?: { retries?: number, preferred?: ContentServerClient}): Promise<T | undefined> {
         try {
-            return await tryOnCluster(execution, cluster, options)
+            return await tryOnCluster(execution, cluster, description, options)
         } catch (error) {
             return undefined
```

Comment:
Should we inform the error here?

Reply:
There is  no need to do so, since the `tryOnCluster` already does that. Here, we are just changing from an error to an `undefined`